cli/cmd/encore/logs.go

```diff
@@ -121,6 +121,6 @@ func streamLogs(appRoot, envName string) {
 
 func init() {
 	rootCmd.AddCommand(logsCmd)
-	logsCmd.Flags().StringVarP(&logsEnv, "env", "e", "", "Environment name to stream logs from (defaults to the production environment)")
+	logsCmd.Flags().StringVarP(&logsEnv, "env", "e", "@primary", "Environment name to stream logs from (defaults to the primary environment)")
```

Comment:
This string is not meaningful to our users, so I think it would be better to keep the default as `""` here and convert it to `@primary` in code